src/controllers/store.js

```diff
@@ -42,10 +42,10 @@ function uploadFile(fileInfo, callback){
 	s3.putObject(params, function(err, data) {
```

Comment:
Should not have data available in function, it isn't being used, which is causing another issue with code climate.